instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+// import Proptypes from 'prop-types';
+
+import './SearchBar.scss';
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      searchQuery: '',
+    }
+  }
+
+  handleInputChange = (event) => {
+    console.log('EVENT', event.target.value)
```

Comment:
Look at all of this extra logic you are using to pass the searchbar value back up to app. Is it necessary to store searchQuery here in local state?